addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -606,6 +606,20 @@
         width: $o-kanban-color-border-width;
         z-index: 1; // show the title over kanban color
     }
+
+    .o_field_monetary {
+        /*rtl:ignore*/
+        direction: ltr;
+    }
+
+    .o_kanban_monetary {
+        /*rtl:ignore*/
+        direction: ltr;
+    }
+
+    span.o_kanban_monetary {
+        display: block;
```

Comment:
maybe it should be inline-block, in the given example block seems ok, but maybe it is or it will be used next to other inline element in which case inline-block seems safer